src/yaml/ingestionTrigger/partData/rasterLayerPartData.yaml

```diff
@@ -0,0 +1,82 @@
+openapi: "3.0.1"
+info:
+  title: Layer polygon part
+  version: 1.0.0
+components:
+  schemas:
+    polygonPartMetadata:
+          type: object
+          description: Layer polygon part data definitions
+          required:
+            - name
+            - imagingTimeBeginUTC
+            - imagingTimeEndUTC
+            - geometry
+            - resolutionDegree
+            - resolutionMeter
+            - sourceResolutionMeter
+            - horizontalAccuracyCE90
+            - sensors
+          properties:
+            id:
```

Comment:
check if this should be mandatory, please remove the "pattern" attribute

Reply:
pattern removed, i'll check with shaziri

Reply:
field currently is not required